File: web3rpc/rpc-specs/code-samples/python/personal/importRawKey.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

privateKey = "cd87934ee007b7a458fa00dc0314fff8b2bd43b3079f46c820c379e483b4fd8e"
passphrase = "mypassword"

w3 = Web3(Web3.HTTPProvider(host))
personal_response = w3.personal.import_raw_key(privateKey, passphrase)

print(personal_response)

File: web3rpc/rpc-specs/code-samples/javascript/personal/importRawKey.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
  const provider = new JsonRpcProvider("https://public-en-baobab.klaytn.net");
  const privateKey =
    "cd87934ee007b7a458fa00dc0314fff8b2bd43b3071f46c820c379e483b4fd8e";
  const passphrase = "gr8=B!0@uc$b";

  provider.personal
    .importRawKey(privateKey, passphrase, {}, (err, data, response) => {})
    .then((data) => {
      console.log(data);
    });
})();
